spec/support/matchers/type/have_nullable_string_matcher.rb

```diff
@@ -33,13 +33,13 @@ def accepts_string?
       def rejects_non_string?
         non_string = 10.0
         @klass.new(@valid_hash.merge(@attribute => non_string))
-      rescue Fortnox::API::InvalidAttributeValueError => error
+      rescue Fortnox::API::AttributeError => e
```

Comment:
It's doing that thing again. We should change this cop ...

Reply:
I'll fix this in an upcoming Rubocop branch...